### PR TITLE
Rename forked pods to Signal app.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

File: SignalServiceKit/src/Util/OWSAsserts.h [hunk 1]
```diff
@@ -119,6 +119,9 @@ NS_ASSUME_NONNULL_BEGIN
 #endif
 
 // This macro is intended for use in Objective-C.
+//
+// Note that we use the "C" flavor of the assert to avoid
+// capturing a strong reference to self.
```
Comment (charlesmchen) [inline SignalServiceKit/src/Util/OWSAsserts.h]: Tiny unrelated change.

**Comment by michaelkirk** [comment]
This is a dupe of the more comprehensive #2984 by @jlund - can we build off of that instead?
